samples/listctrl/listtest.cpp

```diff
@@ -377,6 +379,18 @@ void MyFrame::OnToggleBell(wxCommandEvent& event)
     m_listCtrl->EnableBellOnNoMatch(event.IsChecked());
 }
 
+void MyFrame::OnCheckVisibility(wxCommandEvent& WXUNUSED(event))
+{
+    if( m_listCtrl->IsVisible( 2 ) )
+        wxLogMessage( "Line 2 is visible" );
+    else
+        wxLogMessage( "Line 2 is not visible" );
+    if( m_listCtrl->IsVisible( 9 ) )
```

Comment:
```suggestion
    if ( m_listCtrl->IsVisible(9) )
```

Reply:
@vadz 
I also fixed the function declaration in the docs as I presume this is what the first one is about.